docs/ref/models/relations.txt

```diff
@@ -174,16 +174,16 @@ Related objects reference
 
     .. note::
 
-       Note that ``add()``, ``create()``, ``remove()``, ``clear()``, and
-       ``set()`` all apply database changes immediately for all types of
-       related fields. In other words, there is no need to call ``save()``
-       on either end of the relationship.
-
-       Also, if you are using :ref:`an intermediate model
-       <intermediary-manytomany>` for a many-to-many relationship, then the
-       ``add()``, ``create()``, ``remove()``, and ``set()`` methods are
-       disabled.
-
-       If you use :meth:`~django.db.models.query.QuerySet.prefetch_related`,
-       the ``add()``, ``remove()``, ``clear()``, and ``set()`` methods clear
-       the prefetched cache.
+        Note that ``add()``, ``create()``, ``remove()``, ``clear()``, and
+        ``set()`` all apply database changes immediately for all types of
+        related fields. In other words, there is no need to call ``save()``
+        on either end of the relationship.
+
+        Also, if you are using :ref:`an intermediate model
+        <intermediary-manytomany>` for a many-to-many relationship, then the
+        ``add()``, ``create()``, ``remove()``, and ``set()`` methods are
+        disabled.
+
+        If you use :meth:`~django.db.models.query.QuerySet.prefetch_related`,
+        the ``add()``, ``remove()``, ``clear()``, and ``set()`` methods clear
+        the prefetched cache.
```

Comment:
Can maybe do without this whitespace change?

Reply:
Yes, this shouldn't be part of this patch. If we have an automated way to detect similar mistakes it might be fine to fix them all as a separate patch but I'm not sure that it's worth it.